bears/general/LineCountBear.py

```diff
@@ -10,14 +12,19 @@ class LineCountBear(LocalBear):
     LICENSE = 'AGPL-3.0'
     CAN_DETECT = {'Formatting'}
 
-    def run(self, filename, file, max_lines_per_file: int):
+    def run(self, filename, file, max_lines_per_file: int,
+            min_lines_per_file: int, exclude_blank_lines: bool=False):
         """
         Count the number of lines in a file and ensure that they are
         smaller than a given size.
-
         :param max_lines_per_file: Number of lines allowed per file.
+        :param min_lines_per_file: Minimum lines allowed per file.
+        :param exclude_blank_lines: True if blank lines are to be excluded.
         """
         file_length = len(file)
+        blank_lines = len(list(filter(lambda x: re.match(r'^\s*$', x), file)))
```

Comment:
Two suggestions here:

* Use [re.findall](https://docs.python.org/3.5/library/re.html#re.findall) to get a list of all matches and save yourself the filtering and list creation
* rename the variable to `num_blank_lines` to make it more self explanatory